Classes/Plugin/Eid/SearchInDocument.php

```diff
@@ -59,7 +59,8 @@ public function main(ServerRequestInterface $request)
             $query = $solr->service->createSelect();
             $query->setFields([$fields['id'], $fields['uid'], $fields['page']]);
             $query->setQuery($fields['fulltext'] . ':(' . Solr::escapeQuery((string) $parameters['q']) . ') AND ' . $fields['uid'] . ':' . intval($parameters['uid']));
-            $query->setStart($count)->setRows(20);
+            $query->setStart($start)->setRows(20);
+            $query->addSort('page', $query::SORT_ASC);
```

Comment:
```suggestion
            $query->addSort($fields['page'], $query::SORT_ASC);
```

We should use configured field not string directly. There are possibilities that actually _page_ field is called differently.

Reply:
Very good point. Thank you.